### ISSUE TITLE
registerNotificationOpened is not called when app is in foreground.

### ISSUE THREAD

**Issue by mohitharshan123** [wix/react-native-notifications#1008]
I want to trigger a function when the notification is clicked when the app is running in foreground

**Comment by luca-tomasetti** [comment]
I had the same problem with an Android app, I stopped using this package and moved to this one: https://notifee.app/

**Comment by SethArchambault** [comment]
Notifee doesn't really provide anything for receiving push notifications though right? Looks like they recommend using Firebase Cloud Messaging:
https://notifee.app/react-native/docs/integrations/fcm

**Comment by luca-tomasetti** [comment]
@SethArchambault yes, you are correct. I'm using Notifee together with FIrebase Cloud Messaging.
